includes/class-sensei.php

```diff
@@ -653,29 +727,30 @@ public function initialize_global_objects() {
 
 		} else {
 
-			// Load Frontend Class
+			// Load Frontend Class.
 			$this->frontend = new Sensei_Frontend();
 
-			// Load built in themes support integration
+			// Load built in themes support integration.
 			$this->theme_integration_loader = new Sensei_Theme_Integration_Loader();
 
 		}
 		Sensei_Course_Video_Settings::instance()->init( $this );
 
-		// Load notice Class
+		// Load notice Class.
 		$this->notices = new Sensei_Notices();
 
-		// Load Grading Functionality
+		// Load Grading Functionality.
 		$this->grading = new Sensei_Grading( $this->main_plugin_file_name );
 
-		// Load Email Class
+		// Load Email Class.
 		$this->emails = new Sensei_Emails( $this->main_plugin_file_name );
 
-		// Load Learner Profiles Class
+		// Load Learner Profiles Class.
 		$this->learner_profiles = new Sensei_Learner_Profiles();
 
-		// Load WPML compatibility class
-		$this->sensei_wpml = new Sensei_WPML();
+		// Load WPML compatibility class.
+		$this->sensei_wpml = new WPML();
+		$this->sensei_wpml->init();
```

Comment:
Here is the only meaningful change, the rest is the code style fix.

We instantiate and init a new compatibility class (the old one is deprecated).